cmd/containerd/command/main.go

```diff
@@ -248,6 +248,11 @@ can be used and modified as necessary as a custom configuration.`
 				return fmt.Errorf("failed to get listener for metrics endpoint: %w", err)
 			}
 			serve(ctx, l, server.ServeMetrics)
+		} else {
+			log.G(ctx).Infof("prometheus metrics is disabled,you can specify an address in metrics from config.toml to enable it")
+		}
+		if !config.Metrics.GRPCHistogram {
+			log.G(ctx).Infof("prometheus histogram metrics is disabled,you can set grpc_histogram=true and specify an address in metrics from config.toml to enable it")
```

Comment:
LGTM except for the log message(but I am not good at it, just add my candidate).

- Metrics endpoint does not listen by default, and the metrics address needs to be specified in the Containerd configuration file to enable it.

- gRPC histogram metrics are disabled by default. To turn it on, set grpc_histogram to be true and specify the metrics address in the Containerd configuration file.

Reply:
done,thank you